agave/tasks/sqs_tasks.py

```
@@ -86,12 +102,29 @@ async def run_task(
+                task_metrics = TaskMetrics(
+                    task_name=task_name,
+                    queue_url=queue_url,
+                    concurrent_tasks_counter=lambda: len(BACKGROUND_TASKS),
+                )
+                await emit_metrics(
+                    metrics=task_metrics,
+                    status=log_data['response']['status'],
+                    duration_ms=duration_ms,
+                    metrics_callback=metrics_callback,
+                )
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)